Jenkinsfile

```diff
@@ -33,8 +33,7 @@ metadata:
 spec:
   containers:
   - name: k8s-node
-    image: gcr.io/pso-helmsman-cicd/jenkins-k8s-node:1.1.0
-    imagePullPolicy: Always
```

Comment:
We need the always tag please

Reply:
Can we get the image pull policy back please and I will merge.

Nice job fixing it!!